cgmes/cgmes-shortcircuit/src/test/java/com/powsybl/cgmes/shorcircuit/CgmesImporterTest.java

```diff
@@ -5,16 +5,14 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  * SPDX-License-Identifier: MPL-2.0
  */
-package com.powsybl.cgmes.shortcircuit;
+package com.powsybl.cgmes.shorcircuit;
```

Comment:
Can you create an issue to remember that this package name should be fixed (in another PR, and later)?